packages/cli/src/activate.ts

```diff
@@ -33,7 +33,7 @@ export default async function activate(
     }
   }
 
-  M365TokenProvider.setStatusChangeMap(
+  await M365TokenProvider.setStatusChangeMap(
```

Comment:
Is it possible to just not call `setStatusChangeMap` in CLI? Just:
1. Try set region when activate
2. Try set region whenever login/logout

IMO, during one CLI call, login/logout at most happens only once.